From: Michael Tokarev <mjt@tls.msk.ru>
Subject: spice-common: remove version construction
Date: Thu,  7 Feb 2013 20:52:14 +0400
Message-Id: <1360255935-24510-1-git-send-email-mjt@msgid.tls.msk.ru>
Forwarded: yes

Version string isn't used anywhere in spice-common, and there's
no version for spice-common module per se, either.  Hoever,
configure.ac has this:

 AC_INIT([spice-common],
         [m4_esyscmd(build-aux/git-version-gen .tarball-version)],
         [spice-devel@lists.freedesktop.org])

But since git-version-gen script is not provided in spice-common
module, multiple error messages are produced when generating
configure (running autogen/autoreconf), like this:

  sh: ./build-aux/git-version-gen: not found

(repeated about 50 times).

The following trivial patch removes usage of build-aux/git-version-gen
from AC_INIT and replaces it with a string "noversion", to stop
these scary messages from being produced.

Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
---
 spice-common/configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/spice-common/configure.ac b/spice-common/configure.ac
index c4b0f08..b5cb960 100644
--- a/spice-common/configure.ac
+++ b/spice-common/configure.ac
@@ -1,7 +1,7 @@
 AC_PREREQ([2.63])
 
 AC_INIT([spice-common],
-        [m4_esyscmd(build-aux/git-version-gen .tarball-version)],
+        [noversion],
         [spice-devel@lists.freedesktop.org])
 
 AC_CONFIG_SRCDIR([common/bitops.h])
-- 
1.7.10.4

